doc/src/input/pairs.md

```diff
@@ -0,0 +1,166 @@
+# Pair interactions
+
+## Cutoff treatment
+
+When computing the energy and forces for non-bonded pair interactions, Lumol
+uses a cutoff radius $rc$. This means that the force and energy associated with
+any pair at a distance bigger than $rc$ will be zero. We can use two different
+cutoff scheme, presented in the following section.
+
+In the potentials input file, the cutoff should be specified for all the
+`[[pairs]]` sections. It can be specified once for all the pairs in the `global`
+section, and then overridden for specific interactions:
+
+```toml
+# Use a simple cutoff with a radius of 10 A for all the pairs interactions
+[global]
+cutoff = "10 A"
+
+[[pairs]]
+atoms = ["A", "A"]
+lj = {sigma = "3 A", epsilon = "123 kJ/mol"}
+
+[[pairs]]
+atoms = ["B", "B"]
+lj = {sigma = "3 A", epsilon = "123 kJ/mol"}
+
+# Except for this one, use a shifted cutoff with a radius of 8 A
+[[pairs]]
+atoms = ["A", "B"]
+lj = {sigma = "3 A", epsilon = "123 kJ/mol"}
+cutoff = {shifted = "8 A"}
+```
+
+### Simple cutoff
+
+This scheme just set the potential $U(r)$ to zero for any distance bigger than
+$rc$:
+
+$$ V(r) = \begin{cases}
+    U(r) & r <= rc \\\\
+    0 & r > rc
+\end{cases}$$
+
+In the input, this uses a single string containing the cutoff radius as the
```

Comment:
A suggestion: To use this simple potential truncation, we simply specify a string containing....